### DISCUSSION TITLE
Author information not coming on the Home page of the blog

### DISCUSSION THREAD

**Discussion by puneetg1983** [mmistakes/minimal-mistakes#2577] [Q&A] [answered]
First of all, thanks for an amazing theme. Totally changes the way your blog looks

The issue I am facing is a bit weird. For some reason, I am not able to see the author information section on the home page of the blog - https://puneet-gupta.github.io/

![image](https://user-images.githubusercontent.com/5299838/85281689-0fc07d00-b4a8-11ea-9a19-4ee71cb77b5a.png)

I can see the author information when I go to individual posts but for some reason, this is not showing on the main page. 

The issue happens only on github pages. When I am running the blog locally, everything works fine.

**Comment by mmistakes** [comment, marked as answer]
Could be due to you having two home `index` files. Get rid of one and try again and make sure the one you keep has `author_profile: true` in it's front matter.

![image](https://user-images.githubusercontent.com/1376749/85288507-d403c400-b463-11ea-9851-332bf8e884e7.png)

**Comment by puneet-gupta** [comment]
That was it. Thank you so much :-)
